reference/7/Microsoft.PowerShell.Utility/Select-String.md

```diff
@@ -535,6 +535,29 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -NoEmphasis
+
+By default, the **NoEmphasis** parameter highlights the string found by `Select-String`. To disable
```

Comment:
By default, `Select-String` highlights the string that matches the pattern you search for. The **NoEmphasis** disables this highlighting.

Reply:
Thanks for the feedback! Fixed.